.github/workflows/playwright.yml

```diff
@@ -50,9 +50,13 @@ jobs:
       env:
         PLAYWRIGHT_TEST_BASE_URL: ${{ github.event.deployment_status.target_url }}
       run: yarn playwright test --shard=${{ matrix.shard }}
+    - name: Run Storybook tests
+      env:
+        TARGET_URL: '${{ github.event.deployment_status.target_url }}'
+      run: yarn test-storybook
```

Comment:
@ninaTorgunakova Does this run `yarn storybook` in the background first?

Reply:
No, it doesn't run first, and there would be a high possibility that this step can fail without it. I forgot that we are not publishing it, my bad.
I added missed step, can't test surely without deployment, but now it should be alright. Thanks for your attentiveness!